src/components/Payment/BraintreeDropInInterface.vue

```diff
@@ -53,53 +57,75 @@ export default {
 		paymentTypes: {
 			type: Array,
 			default: () => ['paypal', 'card'],
+			validator(value) {
+				const possiblePaymentTypes = ['card', 'paypal', 'paypalCredit', 'venmo', 'applePay', 'googlePay'];
+				return value.every(elem => possiblePaymentTypes.includes(elem));
+			}
+		},
+		/**
+		 * Drop-In auth options
+		 * Must be 'token-key' or 'client-token'
+		* */
+		auth: {
+			type: String,
+			default: 'client-token',
+			validator(value) {
+				// The value must match one of these strings
+				return ['token-key', 'client-token'].indexOf(value) !== -1;
+			}
 		},
 	},
 	data() {
 		return {
 			btDropinInstance: () => {},
 			clientToken: null,
-			updatingPaymentWrapper: true,
+			updatingPaymentWrapper: false,
```

Comment:
Won't this turn the initial loading state off everywhere?

Reply:
If it actually does need to load -- when (this.auth === 'client-token') and we need a token, then we set loading state before we fetch the token in getDropInToken()